### PR TITLE
sed: refresh/add pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by EmilySeville7cfg** [comment]
@CleanMachine1, CI fails again.

**Comment by CleanMachine1** [comment]
We can ignore it, i think its because the CI is running version of `black` which are affected with the bug.

**Comment by blueskyson** [comment]
The 4th and the 7th example are the same.

File: pages/common/sed.md [hunk 1]
```diff
@@ -1,36 +1,29 @@
...
 
 > Edit text in a scriptable manner.
+> See also: `awk`, `ed`.
 > More information: <https://www.gnu.org/software/sed/manual/sed.html>.
 
-- Replace the first occurrence of a regular expression in each line of a file, and print the result:
+- Execute a specific expression:
 
-`sed 's/{{regular_expression}}/{{replace}}/' {{filename}}`
+`{{command}} | sed '{{s/apple/mango/g}}'`
```
Comment (sbrl) [inline pages/common/sed.md:10]: I think the original was much clearer. Let's revert this change.

**Review by sbrl — COMMENTED** [review]
This is just one example, but changing some of them to specific examples isn't particularly helpful I don't think.

Can you perhaps explain why you think these changes are necessary?

File: pages/common/sed.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/common/sed.md]: Done ;)

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]
